### DISCUSSION TITLE
Unsupported argument error

### DISCUSSION THREAD

**Discussion by kvmallika** [learndevopsonline/learndevopsonline#1504] [Q&A]
Hi,
while trying to create a docdb cluster i am getting the below error,

![image](https://github.com/learndevopsonline/learndevopsonline/assets/130459434/8a874cc8-5bf9-4693-8ff4-434dc52c38db)
main.tf file:
below is the module:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130459434/d287fb00-fcbc-425a-9564-4da5161c7c36)

vars.tf file:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130459434/c343c291-1d4c-4cee-b83a-a0d2d440d683)

**Comment by sreedharm07** [comment]
Hi @kvmallika 
your docdb repo was empty  there is no files in  tf-module-docdb repository (**https://github.com/kvmallika/tf-module-docdb**)
may be you forgot to push the code

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/e0f0e92f-b7c5-466a-a7df-8f0381def55d)


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/e7d7b214-2b03-49a7-b700-f5bd981f67ca)

**Reply by kvmallika** [reply]
Thanks for the reply. I have restarted the Intellij . Now its working fine.
